cmd/clusterctl/api/v1alpha3/provider_type.go

```diff
@@ -46,12 +46,6 @@ type Provider struct {
 	// Version indicates the component version.
 	// +optional
 	Version string `json:"version,omitempty"`
-
-	// WatchedNamespace indicates the namespace where the provider controller is is watching.
-	// if empty the provider controller is watching for objects in all namespaces.
-	// Deprecated: in clusterctl v1alpha4 all the providers watch all the namespaces; this field will be removed in a future version of this API
-	// +optional
-	WatchedNamespace string `json:"watchedNamespace,omitempty"`
```

Comment:
We cannot drop this field. As mentioned above, in my opinion we cannot ever remove fields in released API versions, we can drop them in future versions. (in this case there is no newer version than v1alpha3 for this type)

Reply:
Thanks for pointing that out - I'll revise the PR to keep everything under v1alpha3/4 types the same and only remove types from v1beta1

Reply:
@killianmuldoon I think we have to keep this one (not sure if it's an oversight or it's WIP to keep it)